test/index.spec.js

```diff
@@ -162,14 +162,6 @@ describe('mh', () => {
       ).to.throw(
         /length should be equal/
       )
-
-      const longBuffer = new Buffer(150)
-      longBuffer.fill('a')
-      expect(
-        () => mh.encode(longBuffer, 0x11)
-      ).to.throw(
-        /not yet support/
-      )
```

Comment:
add a test that checks something that is added with more than 127 bytes and that the varint added is correct.